docker/entrypoint.sh

```diff
@@ -6,7 +6,7 @@ COMMAND=$1 && shift 1
 case "$COMMAND" in
   'start-web' )
     echo "Starting Test Server..."
-    yarn start:test-server
+    #yarn start:test-server
```

Comment:
Should we keep this file? (used in the [Dockerfile](https://github.com/standardnotes/snjs/blob/master/Dockerfile))